magento/pwa-studio
# Add to Wishlist from PDP

## Description

Add a product to your wishlist(s) from PDP for both CE and EE.

## Related Issue





Closes PWA-1267.

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


**CE**
1. Go to a PDP without being signed in. You should see no "Add to Favorites" button.
2. Sign in. You should see the button show up.
3. Click the button. It should add to favorites (verify via network/no errors in console) and the button should become disabled with some text like "Added to Favorites"
4. Click some new combination of options. The button should re-enable.

**EE**
1. Go to a PDP without being signed in. You should see no "Add to Favorites" button.
2. Sign in. You should see the button show up.
3. Click the button. It should show a dialog.
4. Select the first list. It should close the dialog if successful.
5. Click the button again after changing some options. 
6. Create a new list with same name as existing. It should fail.
7. Create a new list with a new name. It should add successfully.
8. Keep creating lists until you cannot add any more (10?). Should still let you choose a list to add the item to.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
